### PR TITLE
DEV-3883-feat(issamestring): solution and testing program

### PR DESCRIPTION
**Pull request by mohamedLazyBob**
resolves 01-edu/public#1157

### CODE REVIEW

File: tests/issamestring_test/main.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+func main() {
+	args := [][]string{
+		{`abcd`, `AbCd`},
+		{`abcd`, `abcddd`},
+		{`ab12`, `abcd`},
+		{``, `abcd`},
+		{`ABCD`, `abcd`},
+		{`abcd12HJK`, `abcd12HJK`},
+		{`+++\t----`, `+++\t----`},
+		{`abcdefgh`, `abcdefghijklmnopqrstuvwxyz`},
```
Comment (zainabdnaya) [inline tests/issamestring_test/main.go:19]: Well done , just try to add test that include space and tabulation

**Review by zainabdnaya — COMMENTED** [review]
Code works well!

**Review by jotapero — APPROVED** [review]
Looks Good :alien:

File: solutions/issamestring.go [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package solutions
+
+import (
+	"unicode"
```
Comment (tiagocollot) [inline solutions/issamestring.go:4]: Remove unnecessary "( )" from import.

**Review by tiagocollot — CHANGES_REQUESTED** [review]

File: solutions/issamestring.go (hunk 2 above)
Comment (jotapero) [reply solutions/issamestring.go]: fixed

**Pull request merged**